app/views/layouts/_header.html.erb

```diff
@@ -25,7 +25,7 @@
                 </li>
                 <% if feature_active? :volunteer_ops_list %>
                   <li class="<%= active_if 'volunteer_ops' %>">
-                    <%= link_to "Volunteers", volunteer_ops_path %>
+                    <%= link_to "Volunteers", root_path %>
                   </li>
```

Comment:
@tansaku I'm starting to think that may be this is not the correct fix. I think I should change the `volunteer_ops_path instead`. Right?